### PR TITLE
Choose cip materials

### PR DESCRIPTION
**Pull request by goodviber**
### Context
Trello #567 - We need to add an advisory page to the /core-programme/materials flow
### Changes proposed in this pull request
Addition of static page at /schools/cohorts/2021/core-programme/materials/advisory
### Guidance to review
Sign in as induction tutor, take the "choose CIP" path and page will be visible on clicking "Choose your training materials" link
### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?
See guidance to review above.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,12 +10,40 @@
...
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_04_30_095331) do
+ActiveRecord::Schema.define(version: 2021_05_04_150201) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  create_table "active_storage_attachments", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```
Comment (jamesrose) [inline db/schema.rb:19]: These changes seem unrelated to this PR. Did you intend to commit them?

**Comment by ethanmills** [comment]
The back link on the edit page needs changing

**Comment by ethanmills** [comment]
[CIP Cohort: choose what training materials you want to use](https://trello.com/c/gPUrXdwl/567-cip-cohort-choose-what-training-materials-you-want-to-use)

**Comment by ethanmills** [comment]
[CIP Cohort: choose what training materials you want to use](https://trello.com/c/gPUrXdwl/567-cip-cohort-choose-what-training-materials-you-want-to-use)
